soci/gzip_zinfo.go

```diff
@@ -139,3 +157,26 @@ func (i *GzipZinfo) ExtractData(fileName string, uncompressedSize, uncompressedO
 
 	return bytes, nil
 }
+
+func (i *GzipZinfo) UnmarshalCZinfo() (*CZinfo, error) {
```

Comment:
Suggest renaming this to just `Unmarshal`.

Reply:
Agree.